pkg/storage/test/tuples.go

```diff
@@ -1215,6 +1216,64 @@ func ReadTest(t *testing.T, datastore storage.OpenFGADatastore) {
 	})
 }
 
+func ReadPageTest(t *testing.T, datastore storage.OpenFGADatastore) {
+	ctx := context.Background()
+	storeID := ulid.Make().String()
+
+	tuples := []*openfgav1.TupleKey{
+		tuple.NewTupleKey("document:1", "reader", "user:anne"),
+	}
+
+	err := datastore.Write(ctx, storeID, nil, tuples)
+	require.NoError(t, err)
+
+	tuples = []*openfgav1.TupleKey{
+		tuple.NewTupleKey("document:2", "reader", "user:anne"),
+	}
+
+	err = datastore.Write(ctx, storeID, nil, tuples)
+	require.NoError(t, err)
+
+	tuples = []*openfgav1.TupleKey{
+		tuple.NewTupleKey("document:1", "admin", "user:anne"),
+	}
+
+	err = datastore.Write(ctx, storeID, nil, tuples)
+	require.NoError(t, err)
+
+	t.Run("returns_2_results_when_page_size_2", func(t *testing.T) {
+		tuples, contToken, err := datastore.ReadPage(
+			ctx,
+			storeID,
+			tuple.NewTupleKey("document:1", "", "user:anne"),
+			storage.PaginationOptions{
+				PageSize: 2,
+				From:     "",
+			},
+		)
+		require.NoError(t, err)
+
+		var receivedTuples []*openfgav1.TupleKey
+		for _, tuple := range tuples {
+			receivedTuples = append(receivedTuples, tuple.Key)
+		}
+
+		expectedTuples := []*openfgav1.TupleKey{
+			tuple.NewTupleKey("document:1", "admin", "user:anne"),
+			tuple.NewTupleKey("document:1", "reader", "user:anne"),
+		}
+
+		cmpOpts := []cmp.Option{
+			testutils.TupleKeyCmpTransformer,
+			protocmp.Transform(),
+		}
+		if diff := cmp.Diff(expectedTuples, receivedTuples, cmpOpts...); diff != "" {
+			t.Errorf("mismatch (-want +got):\n%s", diff)
+		}
+		require.Empty(t, contToken)
```

Comment:
We can remove the `if` if we make use of the assert and leverage its ability to provide accurate error messages:

```suggestion
		diff := cmp.Diff(expectedTuples, receivedTuples, cmpOpts...)
		
		assert.Empty(t, diff)
		assert.Empty(t, contToken)
```